styles/designs/carnival/parts/_table-shapes.scss

```diff
@@ -41,9 +41,9 @@
                         )),
                     )
                 )),
+                $Table__BottomBorder--AfterTable,
                 map-merge($Caption__Container--Table, (
                     _components: (
-                        $Table__BottomBorder,
                         $Caption__TitleLabel,
```

Comment:
I changed way of creating bottom border. Previously we added it as a part of caption, but there are some tables that don't have a caption and then border was missing.